pkg/conversion/converter.go

```diff
@@ -670,6 +670,32 @@ func (c *Converter) defaultConvert(sv, dv reflect.Value, scope *scope) error {
 		switch st.Kind() {
 		case reflect.Ptr, reflect.Interface:
 			return c.convert(sv.Elem(), dv.Elem(), scope)
+
+		case reflect.Slice:
```

Comment:
> I wish there was a place more specific to query param encoding/decoding to put this... code this complex in a shared codepath will never get removed :-/

I will gladly accept a pointer to a better spot and I will not hesitate to close this pull in favor of a better one, but I'm pretty sure that sticky code is sticky.  I see these choices:
 1. write a custom converter that is replumbed through scheme to be used sometimes and duplicates nearly the entire conversion path.
 1. try to override "the right" bits some of the time for a special case of "mappy shaped thing".  That's essentially what this does, but you could try duplicating more of the path to see if makes you happier.
 1. write a serializer that takes us to query param shaped bytes and back
 1. ??

Would you like to block on any of those choices instead?